packages/vivaldi-deb/vivaldi-deb.pacscript

```diff
@@ -0,0 +1,7 @@
+name="vivaldi-deb"
+version="4.0.2312.41-1"
+url="https://downloads.vivaldi.com/stable/vivaldi-stable_${version}_amd64.deb"
+description="An advanced browser made with the power user in mind."
+gives="brave-browser-beta"
```

Comment:
```suggestion
gives="vivaldi-stable"
```

I expected you to write the correct `gives` field instead of copy pasting xD

Reply:
Sorry completly forgot to change it after pasting xD

Reply:
You just copy pasted xD